### PR TITLE
ci(wp-desktop): Update Chrome and Chromedriver

### PR DESCRIPTION
**Pull request by scinos**
#### Changes proposed in this Pull Request

* Update Chrome to 87
* Update chromedriver to 11 to match our Electron version

#### Testing instructions

* Verify Desktop e2e tests still pass

### CODE REVIEW

File: .teamcity/_self/projects/DesktopApp.kt [hunk 1]
```diff
@@ -83,9 +83,15 @@ object E2ETests : BuildType({
 				export E2EGUTENBERGUSER="%E2EGUTENBERGUSER%"
 				export E2EPASSWORD="%E2EPASSWORD%"
 				export CI=true
+				export CHROME_VERSION="87.0.4280.141-1"
 
 				# Start framebuffer
-				Xvfb ${'$'}{DISPLAY} -screen 0 1280x1024x24 &
+				sudo Xvfb ${'$'}{DISPLAY} -screen 0 1280x1024x24 &
```
Comment (scinos) [inline .teamcity/_self/projects/DesktopApp.kt:89]: Once this is merged, we can move this to the base docker image.

File: desktop/e2e/tests/e2e.js [hunk 2]
```diff
@@ -19,14 +21,13 @@ const driverConfig = new webdriver.Builder()
 	.usingServer( 'http://localhost:9515' )
 	.setChromeOptions( options )
 	.withCapabilities( {
-		chromeOptions: {
+		'goog:chromeOptions': {
```
Comment (scinos) [inline desktop/e2e/tests/e2e.js:24]: Docs: https://www.electronjs.org/docs/tutorial/using-selenium-and-webdriver#3-connect-to-chromedriver

**Review by wp-desktop — CHANGES_REQUESTED** [review]
WordPress Desktop CI Failure for job "wp-desktop-mac".

@scinos please inspect this job's build steps for breaking changes at [this link](https://circleci.com/gh/Automattic/wp-calypso/1025513). For temporal failures, you may try to "Rerun Workflow from Failed".

Please also ensure this branch is rebased off latest Calypso.

**Comment by scinos** [comment]
This is a comment

**Comment by scinos** [comment]
This is a comment

**Comment by scinos** [comment]
This is a new message

**Comment by scinos** [comment]
nThis is a new message

**Comment by scinos** [comment]
This is a new message

**Comment by scinos** [comment]
This is a new message with "things"
 and
 ## Things

**Comment by scinos** [comment]
This is a new message with "things"
 and
 ## Things

**Comment by scinos** [comment]
This is a new message with "things"
 and
 ## Things

**Comment by scinos** [comment]
Link to live branch: https://calypso.live/?branch=desktop/update-chrome

**Pull request closed without merging**